tests/installation/kubeadm_settings.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Kubic kubeadm role configuration
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    assert_screen 'kubeadm-settings';
+    check_screen 'kubeadm-ntp-empty';    # bsc#1114818
+
+    send_key 'alt-t';
```

Comment:
```suggestion
        send_key 'alt-t';
```

Reply:
I prefer to always set the same ntp server so we can check if config was applied after installation